Capstone/Profile/AccountViewController.m

```diff
@@ -32,6 +32,25 @@ - (void)viewDidLoad {
     self.tableView.dataSource = self;
     [self setUser];
     self->currentUser = [PFUser currentUser];
+    if (self->currentUser == nil) {
+        [self setUser];
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [FBSDKProfile loadCurrentProfileWithCompletion:^(FBSDKProfile * _Nullable profile, NSError * _Nullable error) {
```

Comment:
Let's break this down into a helper function! (just the FB login portion) it will make it easier to understand the viewDidLoad function since you're initializing different parts of the Account page :)